posts/when_to_fold_em.md

```diff
@@ -0,0 +1,48 @@
+# When to fold 'em
```

Comment:
fair point. we need to tell more about our topic in the title. @kjdchapman more than social media interesting/clickbait/whatever, we need to tell people what we're going to ask them to take 5 min of their time for :)

Reply:
The sunk cost fallacy - When to fold an underachieving project ?
?

Reply:
If I was explicitly searching for help on a topic ("argh how do I make Docker work with this CI setup?!") then I'm looking for something matching that description, but if I'm flicking through twitter, I think I'm more likely to click on something weirder.

Possibility I'm probably over-adjusting for my own tastes.

I don't think I have any great suggestions though... what do you think of
> When to fold 'em - how to avoid the sunk cost fallacy

I feel like subtitles would be handy!